eng/pipelines/templates/jobs/vmr-synchronization.yml

```diff
@@ -36,28 +39,28 @@ jobs:
       - group: DotNetBot-AzDO-PAT
       - name: vmrInternalUrl
         value: https://dnceng@dev.azure.com/dnceng/internal/_git/dotnet-dotnet
-  
+
   pool:
     ${{ if eq(variables['System.TeamProject'], 'public') }}:
       name: $(DncEngPublicBuildPool)
-      demands: ImageOverride -equals Build.Ubuntu.2004.Amd64.Open
-    ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+      image: 1es-ubuntu-2004-open
+      os: linux
+    ${{ else }}:
       name: $(DncEngInternalBuildPool)
-      demands: ImageOverride -equals Build.Ubuntu.2004.Amd64
+      demands: ImageOverride -equals 1es-ubuntu-2004
```

Comment:
Shouldn't this use the `image` param instead of `demands` as is done above?

Reply:
Unfortunately not. Somehow each (public and internal) pool needs something else. I found out that specifying both works the best. I won't do it here because this is just a backport so I don't want to diverge from the main version.

Reply:
Oh, I see, thanks for the infp!